### PR TITLE
test: add event test

### PR DESCRIPTION
**Pull request by Han-Ya-Jun**
### Description

Fixes # (issue)

### Checklist

- [x] 填写 PR 描述及相关 issue (write PR description and related issue)
- [x] 代码风格检查通过 (code style check passed)
- [x] PR 中包含单元测试 (include unit test)
- [x] 单元测试通过 (unit test passed)
- [x] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: src/dashboard/apigateway/apigateway/tests/common/event/event.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#  -*- coding: utf-8 -*-
```
Comment (alex-smile) [inline src/dashboard/apigateway/apigateway/tests/common/event/event.py:1]: 文件名，应该为 test_event.py ？

File: src/dashboard/apigateway/apigateway/tests/common/release/publish.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
+#  -*- coding: utf-8 -*-
```
Comment (alex-smile) [inline src/dashboard/apigateway/apigateway/tests/common/release/publish.py:1]: 文件名应该为 test_publish.py ？

**Review by alex-smile — APPROVED** [review]

File: src/dashboard/apigateway/apigateway/tests/common/release/publish.py (hunk 2 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/tests/common/release/publish.py:1]: done

File: src/dashboard/apigateway/apigateway/tests/common/event/event.py (hunk 1 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/tests/common/event/event.py:1]: done

**Review by wklken — COMMENTED** [review]
LGTM

**Review by zhu327 — APPROVED** [review]

**Review by wklken — APPROVED** [review]

**Pull request merged**